data/members/primekey.yaml

```
@@ -0,0 +1,57 @@
+id: primekey
+name: PrimeKey
+slogan: Creating trust for the connected society
+website: https://www.primekey.com/
+
+# Short description, longer content can be placed in `content/members/`
+description: PrimeKey develops open source products for PKI, digital signatures and cryptography
+
+# membership
+memerSince: 2021-04-14
+memberType: Vendor
+workingGroups:
+  - 
+
+# sponsor
+sponsor:
+  level: 
+
+# Blog posts
+blog: 
+  url: https://www.primekey.com/news/
+  feed: 
```

If you include a feed, the most recent new posts will be listed on your profile:
```suggestion
  feed: https://www.primekey.com/news/feed/
```